label_sync/labels.md

```diff
@@ -223,6 +224,12 @@ larger set of contributors to apply/remove them.
 | <a id="kind/proposal" href="#kind/proposal">`kind/proposal`</a> | Issues or PRs related to proposals.| anyone |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
 | <a id="kind/release-blocking" href="#kind/release-blocking">`kind/release-blocking`</a> | Issues or PRs that need to be closed before the next CAPI release| approvers |  [label](https://git.k8s.io/test-infra/prow/plugins/label) |
 
+## Labels that apply to kubernetes-sigs/cluster-api, only for PRs
+
+| Name | Description | Added By | Prow Plugin |
+| ---- | ----------- | -------- | --- |
+| <a id="do-not-merge/needs-area" href="#do-not-merge/needs-area">`do-not-merge/needs-area`</a> | PR is missing an area label| prow |  [require-matching-label](https://git.k8s.io/test-infra/prow/plugins/require-matching-label) |
```

Comment:
I'm not sure how labels here exactly work, so apologies if I write nonsense :grimacing: 

This PR is adding 10 labels under `missingLabels`, and here we mention only `do-not-merge/needs-area`.

Should we mention other labels here as well, or perhaps specify just `do-not-merge/needs-area` under `missingLabels`?

Reply:
FYI - this page is auto-generated from `labels.yaml`. Your question is definitely still relevant there though.